src/lib/UI/Config/Mapper/FieldType/RichText/CustomTag/ChoiceAttributeMapper.php

```diff
@@ -30,7 +30,13 @@ public function mapConfig(
         array $customTagAttributeProperties
     ): array {
         $parentConfig = parent::mapConfig($tagName, $attributeName, $customTagAttributeProperties);
+
         $parentConfig['choices'] = $customTagAttributeProperties['choices'];
+        $parentConfig['choicesLabel'] = [];
+
+        foreach ($parentConfig['choices'] as $choice) {
+            $parentConfig['choicesLabel'][$choice] = "ezrichtext.custom_tags.{$tagName}.attributes.{$attributeName}.choices.{$choice}.label";
```

Comment:
If someone upgrades from 2.5.1 to 2.5.2 will not have those keys in translation. Ideally, we should introduce some fallback to the choice value.

Reply:
Done: https://github.com/ezsystems/ezplatform-admin-ui/pull/1001/commits/1a4a78088baf6a58e06c1395c58caa8bcd082e8f